src/commonMain/kotlin/com/mattprecious/stacker/command/upstack/Onto.kt

```diff
@@ -54,7 +54,7 @@ internal class Onto(
 
 		val operation = Locker.Operation.Restack(
 			startingBranch = currentBranch.name,
-			currentBranch.flattenUp().map { it.name },
+			currentBranch.all.map { it.name }.toList(),
```

Comment:
Yeah this name sucks... we gotta think of something better

Reply:
Naming is haaaaaard.